packages/web/src/app/doc-pages/brand/icon-doc/icon-doc.component.html

```diff
@@ -91,45 +91,6 @@
           og/eller se ikoner som er for små.
         }
       </p>
-      <div>
-        <ul class="e-list">
-          <li>
-            <app-copy [message]="'e-icon--xxl'">
-              <span class="e-text-lg-strong">e-icon--xxl - 56px</span>
-            </app-copy>
-          </li>
-          <li>
-            <app-copy [message]="'e-icon--xl'">
-              <span class="e-text-lg-strong">e-icon--xl - 48px</span>
-            </app-copy>
-          </li>
-          <li>
-            <app-copy [message]="'e-icon--lg'">
-              <span class="e-text-lg-strong">e-icon--lg - 40px</span>
-            </app-copy>
-          </li>
-          <li>
-            <app-copy [message]="'e-icon--md'">
-              <span class="e-text-lg-strong">e-icon--md - 32px</span>
-            </app-copy>
-          </li>
-          <li>
-            <app-copy [message]="'e-icon--sm'">
-              <span class="e-text-lg-strong">e-icon--sm - 24px</span>
-            </app-copy>
-          </li>
-          <li>
-            <app-copy [message]="'e-icon--xs'">
-              <span class="e-text-lg-strong">e-icon--xs - 16px</span>
-            </app-copy>
-          </li>
-          <li>
-            <app-copy [message]="'e-icon--xxs'">
-              <span class="e-text-lg-strong">e-icon--xxs - 8px</span>
-            </app-copy>
-          </li>
-        </ul>
-      </div>
```

Comment:
Was it intentional to delete this list? Personally, I have used it multiple times 👀

Reply:
Since the new method doesn't rely on these classes I removed it, however I kind of agree that it could stay since they still do work. Hmm

Reply:
Added them back hidden behind a little accordion, so that it's still available but doesn't take up as much attention (since I'd recommend people use the `size` attribute instead)

Reply:
Ah, I see what you mean. The t-shirt size -> pixel conversion is the most important in my opinion. I agree that users should use the `size` prop, not the classes.

I would, therefore, suggest a simple conversion list like this:

- xxs: 8px
- xs: 16px
- sm: 24px

etc..